build/params_shared.go

```diff
@@ -2,14 +2,19 @@ package build
 
 import (
 	"math/big"
+
+	"github.com/libp2p/go-libp2p-core/protocol"
+
+	"github.com/filecoin-project/lotus/node/modules/dtypes"
```

Comment:
This makes build depend on `dtypes` which makes whole bunch of things depend on it and dependencies.